src/client/base.ts

```diff
@@ -177,7 +177,13 @@ export default class Base {
         });
         return this.client.post(this.path, 'BaseGetMyGroupVersions', parameters).then((res: base.MyGroupItemsResponse) => {
             const myGroupVersions: base.ItemVersionResultType[] = [];
-            res['my_group_item'].forEach(obj => {
+
+            const myGroupItem = res['my_group_item'];
+            if (myGroupItem == null) {
```

Comment:
```suggestion
            if (myGroupItem === undefined) {
```
あまり `==` 使いたくないので、こんな感じでお願いします！